From a58f82e72578dab89143067fb3dfbf2140e121ef Mon Sep 17 00:00:00 2001
From: caiyinyu <caiyinyu@loongson.cn>
Date: Wed, 12 Jan 2022 16:43:40 +0800
Subject: [PATCH 30/32] update files:

	modified:   manual/math.texi p0
	modified:   sysdeps/loongarch/Implies p12
	modified:   sysdeps/loongarch/nptl/tls.h p5
---
 manual/math.texi             | 2 +-
 sysdeps/loongarch/Implies    | 2 --
 sysdeps/loongarch/nptl/tls.h | 2 +-
 3 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/manual/math.texi b/manual/math.texi
index 477a18b6d1..105d6122df 100644
--- a/manual/math.texi
+++ b/manual/math.texi
@@ -70,7 +70,7 @@ provided for @code{_Float32}, @code{_Float64} and @code{_Float32x} on
 all platforms.
 It is also provided for @code{_Float128} and @code{_Float64x} on
 powerpc64le (PowerPC 64-bits little-endian), x86_64, x86, ia64,
-aarch64, alpha, mips64, riscv, s390 and sparc.
+aarch64, alpha, loongarch, mips64, riscv, s390 and sparc.
 
 @menu
 * Mathematical Constants::      Precise numeric values for often-used
diff --git a/sysdeps/loongarch/Implies b/sysdeps/loongarch/Implies
index c88325b8be..1945b1f4bb 100644
--- a/sysdeps/loongarch/Implies
+++ b/sysdeps/loongarch/Implies
@@ -1,5 +1,3 @@
-init_array
-
 ieee754/ldbl-128
 ieee754/dbl-64
 ieee754/flt-32
diff --git a/sysdeps/loongarch/nptl/tls.h b/sysdeps/loongarch/nptl/tls.h
index 3b17e0b7f5..dd4f70ab0f 100644
--- a/sysdeps/loongarch/nptl/tls.h
+++ b/sysdeps/loongarch/nptl/tls.h
@@ -27,7 +27,7 @@
 #include <stdint.h>
 #include <dl-dtv.h>
 
-register void *__thread_self asm("$tp"); /* FIXME */
+register void *__thread_self asm("$tp");
 #define READ_THREAD_POINTER() ({ __thread_self; })
 
 /* Get system call information.  */
-- 
2.35.1

